app/scripts/modules/core/serverGroup/configure/common/v2instanceArchetypeSelector.directive.js

```diff
@@ -72,14 +71,12 @@ module.exports = angular.module('spinnaker.core.serverGroup.configure.common.v2i
           v2modalWizardService.markComplete('instance-type');
         } catch (e) {
           $log.warn('DEV NOTE: Using deprecated wizard service; consider upgrading to v2 modal wizard');
```

Comment:
This warning doesn't make sense anymore now that there is no deprecated wizard service to use...